habitat/parser.py

```diff
@@ -208,7 +223,15 @@ def _get_config(self, callsign, config=None):
                     .format(config["_id"]))
             return {"id": config["_id"], "payload_configuration": config}
 
-        config = self._find_config_doc(callsign)
+        if (self.rs_config != None) and callsign.startswith(self.rs_prefix):
+            logging.debug("Overriding payload doc lookup for radiosonde telemetry.")
+            config = copy.deepcopy(self.rs_config)
+            # Replace the callsign fields within the config to keep the downstream parsers happy.
```

Comment:
Please keep lines to a max of 79 chars (applies to some other lines in this PR too)

Reply:
Have tried my best here... E501 is not one that I agree with tbh...